Sming/Wiring/IpAddress.cpp

```diff
@@ -44,9 +43,8 @@ size_t IpAddress::printTo(Print& p) const
 
 String IpAddress::toString() const
 {
-	String res;
-    res.reserve(sizeof(address) * 4);
```

Comment:
`reserve` call is an optimisation, should remain `res.reserve(sizeof(ip4_addr_t) * 4);`